string system_id

bool create_new_map
int64 change_map_to_id 
int64 add_kf_id
int64 add_map_point_id 
bool inform_new_big_change
bool clear_current_map
bool clear_atlas
int64 set_map_bad_id
bool remove_bad_maps
bool set_intertial_sensor
bool set_imu_initialized
# int64 set_kf_db_id
# int64 set_orb_vocab_id
